glusterd2/volume/fs_utils.go

```diff
@@ -293,7 +294,8 @@ func MountVolumeBricks(volinfo *Volinfo) error {
 
 	for _, b := range volinfo.GetLocalBricks() {
 		if err := MountBrickDirectory(volinfo, &b, mtab); err != nil {
```

Comment:
`MountVolumeBricks` will also get called while taking the snapshot clone. 
continue during  error scenarios may affect the functionality, @rafikc30 PTAL

Reply:
As @Madhu-1 suggested during a snapshot clone, if the mount failed, then there is no point in continuing the operation. So I would suggest having a rapper function like `RestartMountVolumeBricks` or have a flag passed to the function like
```
func MountVolumeBricks (volinfo *Volinfo, skipError bool) {
        mtab, err := GetMounts()
        if err != nil {
                return err 
        }

        for _, b := range volinfo.GetLocalBricks() {
                if err := MountBrickDirectory(volinfo, &b, mtab); err != nil && !skipError {
                        return err 
                }
        }
   return nil
}
```

Reply:
@rafikc30 if one brick is corrupt why blocking the mount of other bricks(which may belong to other subvolume)?